doc/semantics-of-the-publishing-api.md

```diff
@@ -0,0 +1,406 @@
+that was created with a different `publishing_app`.
+
+The `publishing_app` can then be used to filter content items appropriately when
+requests are made to the Publishing API. The `publishing_app` is also used as a
+means of auditing which applications are making use of the publishing pipeline.
+
+Note: `publishing_app` should be hyphenated as this is the convention used in
+the Router API.
+
+---
+
+### **Conditionally Required**
+
+---
+
+### title
+
+Example: *VAT rates*
+
+Required: Conditionally
+
+The `title` names the content item. It is required except in cases where the
+content item is non-renderable (see [**format**](#format)).
+
+---
+
+### public_updated_at
+
+Example: *2015-01-01T12:00:00Z*
+
+Required: Conditionally
+
+The `public_updated_at` records the last time the content item was updated. This
+is the time that will appear alongside the content item in the front-end to
+inform users of the time at which that particular content item was updated.
+
+The `public_updated_at` is required except in cases where the content item is
+non-renderable (see [**format**](#format)). This will not be set automatically and must be
+provided by the publishing application.
+
+Note: This is subject to change. It may be that we automatically set
+`public_updated_at` on behalf of publishing applications in the future. Please
+speak to the Publishing Platform team if you have questions about this.
+
+---
+
+### rendering_app
+
+Example: *government-frontend*
+
+Required: Conditionally
+
+The `rendering_app` identifies the front-end application that is responsible for
+rendering the content item. The router will use this information to direct
+users' requests to the appropriate front-end application.
+
+The `rendering_app` is required except in cases where the content item is
+non-renderable (see [**format**](#format)).
+
+---
+
+### details
+
+Example: *{ body: "Something about VAT” }*
+
+Required: Conditionally
+
+The `details` (sometimes referred to as “details hash”) contains content and
+other attributes that are specific to the `format` of the content item. The
+[GOV.UK content schemas](https://github.com/alphagov/govuk-content-schemas)
+determine which fields appear in the details and which are required. The details
+can contain arbitrary JSON that will be stored against the content item.
+
+Not all `formats` have required fields and so details is not required unless the
+`format` demands it. If it is not set, it will default to an empty JSON object.
+
+---
+
+### routes
+
+Example: *[{ path: “/vat-rates”, type: "exact" }]*
+
+Required: Conditionally
+
+The `routes` are used to configure the GOV.UK router for the content item. The
+`routes` appear as a JSON array and each element in this array is a JSON object
+that contains the properties *path* and *type*. No other properties are
+supported.  The *type* must be set to *exact* to denote that the route maps to
+an exact path on the GOV.UK website.
```

Comment:
@bradwright I guess my original question was are we supporting the creation of new ones generally rather than technically.  If so we should update the docs to include them as an option, if not we should mention that they used to be a thing but are no longer.